internal/config/canary_config.go

```diff
@@ -16,53 +16,61 @@ import (
 
 const (
 	// environment variables declaration
-	BootstrapServersEnvVar       = "KAFKA_BOOTSTRAP_SERVERS"
-	TopicEnvVar                  = "TOPIC"
-	ReconcileIntervalEnvVar      = "RECONCILE_INTERVAL_MS"
-	ClientIDEnvVar               = "CLIENT_ID"
-	ConsumerGroupIDEnvVar        = "CONSUMER_GROUP_ID"
-	TLSEnabledEnvVar             = "TLS_ENABLED"
-	ProducerLatencyBucketsEnvVar = "PRODUCER_LATENCY_BUCKETS"
-	EndToEndLatencyBucketsEnvVar = "ENDTOEND_LATENCY_BUCKETS"
-	ExpectedClusterSizeEnvVar    = "EXPECTED_CLUSTER_SIZE"
+	BootstrapServersEnvVar            = "KAFKA_BOOTSTRAP_SERVERS"
+	BootstrapBackoffMaxAttemptsEnvVar = "KAFKA_BOOTSTRAP_BACKOFF_MAX_ATTEMPTS"
+	BootstrapBackoffScaleEnvVar       = "KAFKA_BOOTSTRAP_BACKOFF_SCALE"
+	TopicEnvVar                       = "TOPIC"
+	ReconcileIntervalEnvVar           = "RECONCILE_INTERVAL_MS"
+	ClientIDEnvVar                    = "CLIENT_ID"
+	ConsumerGroupIDEnvVar             = "CONSUMER_GROUP_ID"
+	TLSEnabledEnvVar                  = "TLS_ENABLED"
+	ProducerLatencyBucketsEnvVar      = "PRODUCER_LATENCY_BUCKETS"
+	EndToEndLatencyBucketsEnvVar      = "ENDTOEND_LATENCY_BUCKETS"
+	ExpectedClusterSizeEnvVar         = "EXPECTED_CLUSTER_SIZE"
```

Comment:
Can we avoid aligning these? I know it looks a bit prettier, but as in this case it makes for much noisier diffs.

Reply:
@tombentley I would be with you but it's Go stuff and it's good practice in the Go land. Each IDE align in this way automatically because of this.